torch/multiprocessing/reductions.py

```diff
@@ -152,6 +153,11 @@ def rebuild_meta_tensor(
     return t
 
 
+@_deprecated(
+    "`torch.multiprocessing.reductions.rebuild_cuda_tensor` is deprecated."
+    "Please use `torch.multiprocessing.reductions.rebuild_device_tensor` instead.",
+    category=FutureWarning,
+)
```

Comment:
Actually, I take it back, this is multiprocessing reductions, these should have less stringent BC reqs. Harmless to keep it as is though...